x-pack/plugins/security_solution/public/flyout/left/components/analyze_graph.tsx

```diff
@@ -7,16 +7,55 @@
 
 import type { FC } from 'react';
 import React from 'react';
-import { EuiText } from '@elastic/eui';
+import { EuiEmptyPrompt } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { useLeftPanelContext } from '../context';
 import { ANALYZER_GRAPH_TEST_ID } from './test_ids';
+import { Resolver } from '../../../resolver/view';
+import { useTimelineDataFilters } from '../../../timelines/containers/use_timeline_data_filters';
+import { ERROR_TITLE, ERROR_MESSAGE } from '../../shared/translations';
+import { isActiveTimeline } from '../../../helpers';
 
 export const ANALYZE_GRAPH_ID = 'analyze_graph';
 
+const ANALYZER = i18n.translate('xpack.securitySolution.flyout.analyzerErrorTitle', {
+  defaultMessage: 'analyzer',
+});
+
 /**
  * Analyzer graph view displayed in the document details expandable flyout left section under the Visualize tab
  */
 export const AnalyzeGraph: FC = () => {
-  return <EuiText data-test-subj={ANALYZER_GRAPH_TEST_ID}>{'Analyzer graph'}</EuiText>;
+  const { eventId } = useLeftPanelContext();
+  const scopeId = 'fly-out';
+  const { from, to, shouldUpdate, selectedPatterns } = useTimelineDataFilters(
+    isActiveTimeline(scopeId)
+  );
+
+  if (!eventId) {
+    return (
+      <EuiEmptyPrompt
+        iconType="error"
+        color="danger"
+        title={<h2>{ERROR_TITLE(ANALYZER)}</h2>}
+        body={<p>{ERROR_MESSAGE(ANALYZER)}</p>}
+        data-test-subj={ANALYZER_GRAPH_TEST_ID}
```

Comment:
I think for clarity I would use a different `data-test-subj` here: `ANALYSER_GRAPH_ERROR_TEST_ID`. Because we're using these in jest tests as well as Cypress tests, it would be easy to think the test would be successful because both the analyser graph and its error states have the same `data-test-subj`